domain/src/main/java/life/qbic/domain/usermanagement/User.java

```diff
@@ -55,7 +55,8 @@ protected User() {}
    */
   public static User create(String fullName, String email) throws UserException {
     String uuid = String.valueOf(UUID.randomUUID());
-    var user = new User(fullName);
+    var user = new User();
+    user.setFullName(fullName);
```

Comment:
Why did you change that?

Reply:
I would prefer to keep it as it was since it is no real policy to check for the presence of a value.